keyboards/nibiria/stream15/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
This file should be removed, `DYNAMIC_KEYMAP_LAYER_COUNT ` isnt required for such a small keyboard.